src/www/ui/api/Controllers/JobController.php

```diff
@@ -40,6 +41,19 @@
  */
 class JobController extends RestController
 {
+  public function getAllJobs($request, $response, $args)
+  {
+    $limit = $request->getHeaderLine('limit');
+    $page = $request->getHeaderLine('page');
```

Comment:
Consider adding the filters/checks for these variables.

https://github.com/fossology/fossology/blob/5eaca363ca51c212038939acc98c24672b6392e0/src/www/ui/api/Controllers/JobController.php#L75-L90

Reply:
in `JobControlersTest.php` right??

Reply:
Resolved